### PR TITLE
Automate release cut

### PR DESCRIPTION
**Pull request by stafot**
Add also arm64 binaries and arm64 containers
Issue: MM-40777

#### Summary
Automate release cut and notes.
Add arm64 binaries and arm64 containers.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-40777

#### Release Note

```release-note
Automate release cut and notes.
Add arm64 binaries and arm64 containers.
```

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -79,6 +79,16 @@ jobs:
...
         destination: security-scan
 
+  release:
+    executor: default
+    steps:
+      - checkout
+      - run:
+          command: |
+            sudo apt update && sudo apt install hub git
+            GO111MODULE=on go get k8s.io/release/cmd/release-notes
```
Comment (spirosoik) [inline .circleci/config.yml:89]: I think you don't need `GO111MODULE` as the image is golang and will run under the golang paths

Comment (spirosoik) [reply .circleci/config.yml:89]: also for all the non-make commands just add them in a separate `Makefile` command

File: Makefile [hunk 2]
```diff
@@ -51,19 +51,34 @@ govet:
...
-build: ## Build the cloud-db-factory-vertical-scaling
+build:
 	@echo Building Cloud-DB-Factory-Vertical-Scaling
-	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 $(GO) build -gcflags all=-trimpath=$(PWD) -asmflags all=-trimpath=$(PWD) -a -installsuffix cgo -o build/_output/bin/main  ./
+	GOOS=linux CGO_ENABLED=0 $(GO) build -gcflags all=-trimpath=$(PWD) -asmflags all=-trimpath=$(PWD) -a -installsuffix cgo -o build/_output/bin/main  ./
 
 build-image:  ## Build the docker image for cloud-db-factory-vertical-scaling
 	@echo Building Cloud-DB-Factory-Vertical-Scaling Docker Image
-	docker build \
+	docker buildx build \
```
Comment (spirosoik) [inline Makefile:69]: ❤️

**Review by spirosoik — COMMENTED** [review]
minor few comments

**Review by stylianosrigas — APPROVED** [review]
LGTM

**Review by spirosoik — APPROVED** [review]

**Pull request merged**